filament/backend/src/Program.cpp

```diff
@@ -91,6 +91,12 @@ Program& Program::specializationConstants(
     return *this;
 }
 
+Program& Program::pushConstants(ShaderStage stage,
+        utils::FixedCapacityVector<char const*> constants) noexcept {
+    mPushConstants[static_cast<uint8_t>(stage)] = constants;
```

Comment:
you need to `std::move()` or `std::swap()` here

Reply:
done